src/ScottLogic.Internal.Training/ScottLogic.Internal.Training.Matcher/Trade.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace ScottLogic.Internal.Training.Matcher
+{
+    public class Trade : IEquatable<Trade>
+    {
+        private int AccountNumber;
```

Comment:
If Account number is a private field rather than a property then I would name it _accountNumber.

The naming conventions for C# tends to be for fields _fieldName and for properties PropertyName